AmplifyPlugins/Auth/Tests/AWSCognitoAuthPluginUnitTests/OperationTests/ClientBehaviorTests/AWSCognitoAuthClientBehaviorTests.swift

```diff
@@ -0,0 +1,115 @@
+//
+// Copyright Amazon.com Inc. or its affiliates.
+// All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import Foundation
+
+import XCTest
+import AWSCognitoIdentity
+@testable import Amplify
+@testable import AWSCognitoAuthPlugin
+import AWSCognitoIdentityProvider
+
+class AWSCognitoAuthClientBehaviorTests: XCTestCase {
```

Comment:
Lets move all these test here and make sure the result are same - https://github.com/aws-amplify/amplify-ios/blob/main/AmplifyPlugins/Auth/AWSCognitoAuthPluginTests/AuthenticationProviderTests/AuthenticationProviderResendSignupCodeTests.swift